crunchy/cli/auto_cmd.py

```diff
@@ -57,7 +57,8 @@ def fastq(ctx, indir, spring_path, first, second, dry_run):
 
     if indir:
         LOG.info(
-            "This will recursively compress and delete fastqs in %s", indir,
+            "This will recursively compress and delete fastqs in %s",
```

Comment:
is formatting this str out of scope?

Reply:
I would say so - focusing on the security issue.